### ISSUE TITLE
[Question] Are Wink Shortcuts supported?

### ISSUE THREAD

**Issue by pixelrogue1** [KraigM/homebridge-wink#95]
[Question] Are Wink Shortcuts supported?

Have a number of shortcuts that would be great to speak to Siri, however they are not appearing. Better still, not finding a way to replicate the shortcuts themselves through HomeKit.

Objective:  "Hey Siri, Goodnight."  
Goodnight launches Wink shortcut called Goodnight, which kicks off locking doors, turning out lights and such. This is something I would think as simple for HomeKit and not sure what I'm missing or maybe I am not thinking about it the right way. 

HomeKit automations require something solid (enter/exit, sensor, change of status to a device) - no way to create something with only a button on the screen, or Siri command. What am I missing?

**Comment by benjackson1111** [comment]
I have this same issue. Shortcuts are the only way to make 'scenes' in Wink and they don't appear in Homekit, and there doesn't seem to be any mention of it anywhere as to why Shortcuts aren't supported or what the status of them is or how it's handled in the plugin etc. If I try to use Homekit scenes with the lights, they don't work properly. Colors don't work correctly and there's delays, but the worst thing is that a scene with these wink connected lights, you say pure white and it does some weird color, or ask for blue and it's red. And I can't get full brightness out of the lights even if Homekit says to go 100%. These are the Osram lights (they're good but not if you wanna use Siri, just better to stick with Philips everywhere if you have the money and proper fittings, I couldn't because some guy installed modern recessed lights and Philips doesn't make a recessed light integrated product).

To HELP you guys for now, the workaround is to use IFTTT and iOS Shortcuts app to do a web trigger, however it can take 10-120 seconds for the wink shortcut to trigger this way.  At least the homekit shortcut will be triggering a REAL wink shortcut, which gives you access to all the colors and full brightness. So for pixelrogue1, if you have a cool shortcut in wink to do a bunch of stuff and you don't need it to be IMMEDIATE, try using iOS shortcut to trigger an IFTTT web hook, and that'll then activate the wink shortcut you want. It's not too bad!

Still, would love native support of Wink Shortcuts in the plugin!! Thank you nonetheless for this plugin. :)

**Comment by pixelrogue1** [comment]
HomeBridge works. I can get Siri working 90% with my z-wave functionality. It will lock, but not unlock, doors. Everything else seems to work(albeit I haven’t done thorough testing with all devices - anything we care about works but unlocking.

Wink is about to die it seems on its last legs.

**Comment by benjackson1111** [comment]
Yeah homebridge mainly works for me just not my lighting via wink. I have a ton of stuff working via homebridge on the pi. I didn't know Wink was dying, i specifically bought it about 6 months ago to be a better hub for these damn osram/sylvania lights. The osram hub was even worse in terms of delays or not even working, at least the wink is reliable and doesn't give up. 

I'm moving on to investigate getting homekit to tell ALEXA to activate scenes (because Alexa DOES import wink shortcuts as scenes). Then I wouldn't have the delay or the roundtrip via the IFTTT cloud.

Thanks for replying! But yeah, if you ever get stuck and wanna use a wink shortcut, the iFTTT route isn't too bad.

**Comment by AppleTechy** [comment]
@pixelrogue1  Please see this plugin for homebridge as it is the most update to date version of the Wink Plugin https://github.com/sibartlett/homebridge-wink3

**Comment by pixelrogue1** [comment]
Thank you for bringing this to my attention!!
I just upgraded servers and need to get back to reinstalling HomeBridge. Maybe this will let me unlock doors (something the previous version missed)
